gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -507,10 +507,12 @@ MM_ParallelGlobalGC::mainThreadGarbageCollect(MM_EnvironmentBase *env, MM_Alloca
 #if defined(OMR_GC_MODRON_COMPACTION)
 			if (compactedThisCycle) {
 				OMRPORT_ACCESS_FROM_ENVIRONMENT(env);
+				Trc_MM_FixHeapForWalk_Entry(env->getLanguageVMThread(), FIXUP_DEBUG_TOOLING);
```

Comment:
hm, slight abuse of these 2 existing trace points (reporting reason, where memory type was expected)
I think the point was to say who did fixup (compactor, rather Global GC), not so what was the reason for it (tooling or whatever else). Well, it would be nice to report reason too, but the trace point was not doing it, so let's settle on that.

Feel free to introduce new ones in Compactor's fixHeapForWalk, and to be consistent, they should not report reason.

I see that classunloading fixup has its own traced point and does not report the reason, either.

Reply:
for consistency, we could pass memory type and reason to Compactor's fixHeapForWalk, and then set walk time and reason there

Reply:
Ok. I'm quite sure I understand correctly. I will push changes shortly to ensure this is the case.